lib/midcom/helper/datamanager2/widget/tinymce.php

```diff
@@ -267,6 +263,33 @@ private function _get_tiny_configuration ()
 plugins : ["table save contextmenu link fullscreen image imagetools"],
 toolbar: "bold italic | bullist | link image | code fullscreen",
 extended_valid_elements : "a[name|href|target|title|onclick],img[class|src|border=0|alt|title|hspace|vspace|width|height|align|onmouseover|onmouseout|name|style],hr[class|width|size|noshade],font[face|size|color|style],span[class|align|style]",
+{$this->_get_imagetools_configuration()}
+        
+EOT;
+    }
+
+    /**
+     * Returns the imagetools configuration
+     */
+    private function _get_imagetools_configuration()
+    {
+        $hostname = midcom::get()->get_host_name();
+        $guid = null;
+        if(empty($this->_type->storage->object) == true)
```

Comment:
you can just write `if (empty($this->_type->storage->object))`, there's no need to compare with `true`